templates/repo/release/list.tmpl

```diff
@@ -53,21 +53,24 @@
 							<div class="download">
 								<h2>{{$.i18n.Tr "repo.release.downloads"}}</h2>
 								<ul class="list">
+									{{$tagname := .TagName}}
+									{{if .Attachments}}
+										{{range $attachment := .Attachments}}
+										<li>
+											<a target="_blank" rel="noopener" href="{{$.RepoLink}}/releases/download/{{$tagname}}/{{$attachment.Name}}">
+												<span class="ui image octicon octicon-desktop-download" title='{{$attachment.Name}}'></span> {{$attachment.Name}} 
+												<a target="_blank" href="{{$.RepoLink}}/releases/download/{{$tagname}}/{{$attachment.Name}}" class="ui right"><span class="ui image octicon octicon-file-binary" title='{{$attachment.Name}}'></span>{{$attachment.FileSize}}</a>
```

Comment:
There should not be <a> tag inside <a> tag. File size shoud be moved out of link, seperated with space and placed in brackets: `</a> ({{$attachment.FileSize}})`